flytekit/models/core/workflow.py

```diff
@@ -381,7 +381,9 @@ def from_flyte_idl(cls, pb2_object: _core_workflow.GateNode) -> "GateNode":
 
 
 class ArrayNode(_common.FlyteIdlEntity):
-    def __init__(self, node: "Node", parallelism=None, min_successes=None, min_success_ratio=None) -> None:
+    def __init__(
+        self, node: "Node", parallelism=None, min_successes=None, min_success_ratio=None, execution_version=None
```

Comment:
this will be utilized for determining how propeller will handle the execution (mainly eventing and storing state) of an ArrayNode. Have the IDL change made on the branch in our private fork. Will merge the IDL change into OSS